packages/tasit-contracts/src/helpers/removeFunctionsFromABI.js

```diff
@@ -20,6 +20,7 @@ const removeFunctionsFromABI = (contractJsonPath, removalCondition) => {
   fs.writeFileSync(contractJsonPath, data);
 };
 
+// eslint-disable-next-line no-unused-vars
 const removeFromEstateRegistry = () => {
```

Comment:
What's the reason this is unused right now?

Reply:
There is a comment on top of the file:

```javascript
// Remove overloaded functions
// This helper isn't be used right now, but it'll be useful on that issue:
// https://github.com/tasitlabs/TasitSDK/issues/241 (step 3)
```

Reply:
Resolving this since we can continue the discussion in the other similar thread